books/free-programming-books-subjects.md

```diff
@@ -519,6 +519,7 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
 ### Mathematics For Computer Science
 
 * [A Mathematical Theory of Communication](https://archive.org/details/bstj27-4-623) - Claude E.Shannon
+* [Combinatorial Problems And Exercises](https://archive.org/details/in.ernet.dli.2015.141538/page/n25/mode/2up) - L. Lovasz *(:card_file_box: archived)*
```

Comment:
why are you linking to page 29? Please find a link for the book, not a page in a book

Reply:
Using most authoritative url linking to bookcover :wink:

```suggestion
* [Combinatorial Problems And Exercises (1979)](https://archive.org/details/in.ernet.dli.2015.141538/) - L. Lovasz *(:card_file_box: archived)*
```